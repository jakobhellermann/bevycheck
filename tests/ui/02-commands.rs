#![allow(unused)]
use bevy::prelude::*;

#[bevycheck::system]
fn system(_: &mut Commands) {}

fn main() {
    system.system();
}
