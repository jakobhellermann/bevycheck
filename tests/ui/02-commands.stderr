error: invalid system parameter
 --> tests/ui/02-commands.rs:5:14
  |
5 | fn system(_: &mut Commands) {}
  |              ^^^^^^^^^^^^^
  |
  = help: use `mut commands: Commands`
